prow/jobs/jinja/presubmits/service_tests.jinja2

```diff
@@ -32,7 +32,45 @@
           value: {{ service }}
         - name: DUMP_CONTROLLER_LOGS
           value: "true"
+        - name: ONLY_TEST_E2E
+          value: "true"
         command: ["wrapper.sh", "bash", "-c", "make kind-test SERVICE=$SERVICE"]
+
+  - name: {{ service }}-release-test
+    decorate: true
+    optional: false
+    always_run: true
+    labels:
+	  preset-dind-enabled: "true"
+	  preset-kind-volume-mounts: "true"
+    extra_refs:
+    - org: aws-controllers-k8s
+	  repo: code-generator
+	  base_ref: main
+	  workdir: false
+    - org: aws-controllers-k8s
+	  repo: test-infra
+	  base_ref: main
+	  workdir: true
+    spec:
+	  serviceAccountName: pre-submit-service-account
+	  containers:
+	  - image: public.ecr.aws/m5q3e4b2/prow:prow-integration-0.0.4
+	    securityContext:
+		  privileged: true
+	    resources:
+		  requests:
+		    cpu: 2
+		  limits:
+		    cpu: 8
```

Comment:
Hmmm I'll monitor these. I'm worried we are going to blow up our limits with requests this large. For future me, though.